include/circt/Dialect/Verif/Passes.td

```diff
@@ -42,4 +42,17 @@ def VerifyClockedAssertLike : Pass<"verify-clocked-assert-like", "hw::HWModuleOp
   let constructor = "circt::verif::createVerifyClockedAssertLikePass()";
 }
 
+/**
+ * Converts verif.formal ops into hw.module to prepare for export verilog.
+ */
+def PrepareVerifForExportVerilog : 
+    Pass<"prepare-verif-for-export-verilog", "mlir::ModuleOp"> {
+  let summary = "Prepare verif for verilog export by lowering to hw dialect.";
+  let description = [{
+    Converts verif.formal ops into hw.module to reuse the export verilog module
+    logic.
+  }];
+  let constructor = "circt::verif::createPrepareVerifForExportVerilogPass()";
```

Comment:
Consider dropping this line. TableGen will auto-gen the constructor for you then.

Reply:
You'll want to change the def name to `PrepareVerifForExportVerilogPass` as well, otherwise the function created will be called `createPrepareVerifForExportVerilog()`. 😁